### DISCUSSION TITLE
Replace on conflict when saving links

### DISCUSSION THREAD

**Discussion by dev-thinks** [isar/isar#304] [Help]
Is there an option to specify the "replaceOnConflict" to true when saving a link that is having an unique constraint?

**Comment by simc** [comment]
Hey, Could you give an example how this would be used?

**Reply by ghost** [reply]
For example, the linked class is already in database. When save the relation, it will throw exception. 

It should be able to override or create a link, like the `put` method.

https://github.com/isar/isar/discussions/331#discussioncomment-2547732

**Reply by ghost** [reply]
> For example, the linked class is already in database. When save the relation, it will throw exception. 
> 
> It should be able to override or create a link, like the `put` method.
> 
> https://github.com/isar/isar/discussions/331#discussioncomment-2547732

Is the latest `update` method on links for this kind of usage?

It seems not working with

```
await object.links.update(...)


await object.links.save();
```
